tbx/core/blocks.py

```diff
@@ -308,6 +308,46 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/four_photo_collage_block.html"
 
 
+class KeyPointIconChoice(models.TextChoices):
+    CALENDAR = "key-calendar", "calendar icon"
+    CONVERSATION = "key-conversation", "chat bubbles icon"
+    LIGHTBULB = "key-lightbulb", "lightbulb icon"
+    MAIL = "key-mail", "mail icon"
+    MEGAPHONE = "key-megaphone", "megaphone icon"
+    PEOPLE = "key-people", "people icon"
+    BULLSEYE = "key-bullseye", "target icon"
+    UP_ARROW = "key-up-arrow", "up arrow icon"
+
+
+class IconKeyPointBlock(blocks.StructBlock):
+    icon = blocks.ChoiceBlock(
+        choices=KeyPointIconChoice.choices,
+        default=KeyPointIconChoice.LIGHTBULB,
+        max_length=32,
+    )
+    icon_label = blocks.CharBlock()
+    heading = blocks.CharBlock()
+    description = blocks.RichTextBlock(features=settings.NO_HEADING_RICH_TEXT_FEATURES)
+
+    class Meta:
+        icon = "breadcrumb-expand"
+
+
+class IconKeyPointsBlock(blocks.StructBlock):
```

Comment:
This one doesn't seem to render properly for me, it's not picking up the template :thinking:

Reply:
I originally didn't add in the icons yet because I wasn't sure if we were doing animated icons– which would require a different approach.

I've added them in now in [e8ccfb5](https://github.com/torchbox/torchbox.com/pull/336/commits/e8ccfb5f22410ff765e00fd0de37520700a5095e)